From: Sergio Martins <iamsergio@gmail.com>
Date: Tue, 28 May 2013 20:22:11 +0000
Subject: Compile with boost-1.53
X-Git-Url: http://quickgit.kde.org/?p=akonadi.git&a=commitdiff&h=fe3f49cbd21a634c3257903a229b7d5357ee194a
---
Compile with boost-1.53

The bug isn't going to be fixed in Qt4, so we need the ifdef.

CCMAIL: vkrause@kde.org
---


--- a/shared/akapplication.h
+++ b/shared/akapplication.h
@@ -23,7 +23,9 @@
 #include <QtCore/QObject>
 
 #ifndef _WIN32_WCE
-#include <boost/program_options.hpp>
+# ifndef Q_MOC_RUN
+#  include <boost/program_options.hpp>
+# endif
 #endif
 
 class QCoreApplication;

